codex/sales.nim

```diff
@@ -35,13 +34,12 @@ import ./sales/states/unknown
 ##     |                          | ---- storage proof ---> |
 
 export stint
-export availability
+export reservations
```

Comment:
Should we also replace "availability" with "reservations" in the comment describing the Sales mechanism above?

Reply:
I think availability is still accurate here as availability is added via the REST api as a first step.